crates/node/src/interactive_cli/identity.rs

```diff
@@ -125,8 +125,8 @@ impl IdentityCommand {
     }
 }
 
-fn list_identities(node: &Node, context: Option<Alias<ContextId>>, ind: &str) -> EyreResult<()> {
-    let context_id = if let Some(ctx) = context {
+fn list_identities(node: &Node, context: Alias<ContextId>, ind: &str) -> EyreResult<()> {
+    let context_id = if let ctx = context {
```

Comment:
this is an infallible pattern match match, but anyway factoring your comment for `meroctl` we can revert to what you had before, since it'll be covered by the listing effort

https://github.com/calimero-network/core/pull/1171#discussion_r2039984637

Reply:
this is unresolved

to clarify, infallible / irrefutable here means the `else` block will never run because you replaced `Some(ctx)` for `ctx` which is unconditional